### DISCUSSION TITLE
Response is wrapped in quotes?

### DISCUSSION THREAD

**Discussion by ChronoDK** [fastapi/fastapi#2867] [Questions] [answered]
Hi,

This is the return statement of my get request:
`return base64.b64encode( user.json().encode('utf-8') )`

When the result is received on the client, it is wrapped in quotes? This forces me to replace the quotes before base64 decoding, but I wonder if could get around that? It seems strange to me that these quotes are added.

**Comment by HarrySky** [comment, marked as answer]
AFAIK by default responses are converted to JSON, and since you return bytes string - it will be represented in JSON as quoted string:
```python
# essentially behind the scenes it works like this
import json
from base64 import b64encode
your_response = b64encode( "{'id': 1}".encode('utf-8') )
json.dumps(str(your_response, "utf-8"))
# Returns properly encoded JSON string - '"eydpZCc6IDF9"'
```

Try returning `fastapi.responses.PlainTextResponse` from endpoint instead, and read docs next time, default JSON response is mentioned there.

**Reply by ChronoDK** [reply]
Thanks, you are correct! Very cool actually. 

The docs are amazing btw, not sure why I missed this.

**Reply by ghandic** [reply]
great explaination @HarrySky , @ChronoDK did that solve your issue? If it did could you mark it as the answer to help out others coming with the same issue 😄
